### ISSUE TITLE
Update bounding box calculation

### ISSUE THREAD

**Issue by samsoe** [samsoe/mpg_aerial_survey#31]
I think there is still a small error with how the bbox is defined. Take a close look here:
```
bbox = box(
            point.x - side_len_m, #establishes left limit
            point.y - side_len_m, #establishes bottom limit
            point.x + side_len_m, #establishes right limit
            point.y + side_len_m, #establishes top limit
        )
```
... so  to, eg, establish the left limit it sets the side of the box side_len_m left of the ground truth point. So, by default, it will set the left side a meter left, and all others sides will be similarly established.
If you want a 1m x 1m box by default, you'll achieve expected behavior with something like this:
```
half_side = side_len_m / 2  # Calculate half the side length
    bounding_boxes = []
    for point in gdf.geometry:
        bbox = box(
            point.x - half_side,
            point.y - half_side,
            point.x + half_side,
            point.y + half_side,
        )
```

**Comment by samsoe** [comment]
Issue addressed in f919cb4

**Comment by samsoe** [comment]
Issue addressed in [dadec81](https://github.com/samsoe/mpg_aerial_survey/commit/dadec816e357bf419e6caadbd1603a322a4fb66d)
